bindings/wasm/examples/src/revoke_vc.js

```diff
@@ -43,8 +44,17 @@ async function revokeVC(clientConfig) {
     console.log(`Explore the Issuer DID Document: ${clientConfig.explorer.resolverUrl(issuer.doc.id)}`);
 
     // Check the verifiable credential
-    const result = await client.checkCredential(signedVc.toString(), VerifierOptions.default());
-    console.log(`VC verification result (false = revoked): ${result.verified}`);
+    try {
+        const resolver = await new Resolver();
+        await resolver.verifyCredential(
+            signedVc, 
+            CredentialValidationOptions.default(),
+            FailFast.Yes
+            ); 
+    } catch (exception) {
+    console.log(`${exception.message}`)
+    console.log(`Credential successfully revoked!`);
+    }
```

Comment:
Similar to the `merkle_key` example this should throw an error if the credential was not revoked correctly.